pkg/sql/logictest/testdata/logic_test/set_role

```diff
@@ -338,10 +340,28 @@ SELECT current_user, session_user()
 ----
 testuser  root
 
+query T
+SELECT user_name FROM crdb_internal.node_sessions
+WHERE active_queries LIKE 'SELECT user_name%'
+----
+root
+
 statement ok
 ROLLBACK
 
 query TT
 SELECT current_user, session_user()
 ----
 root  root
+
+statement ok
+SET LOCAL ROLE testuser
```

Comment:
isn't this a no-op anyway since it's a SET LOCAL outside of a txn? or was it intended to be SET ROLE? (if so, maybe we might as well add a test for SET LOCAL ROLE inside a txn as well)

Reply:
d'oh, yep!